kiwi/container/__init__.py

```diff
@@ -30,9 +30,9 @@ class ContainerImage(object):
     :param string name: container system name
     :param string root_dir: root directory path name
     """
-    def __new__(self, name, root_dir):
+    def __new__(self, name, root_dir, custom_args=None):
```

Comment:
`custom_args` are not used in the constructor so they are always `None,` see `ContainerBuilder.create` method. I guess a map of all `containerconfig` attributes should be included as `custom_args.`